lightwood/encoder/numeric/numeric.py

```diff
@@ -1,46 +1,63 @@
 import math
+from typing import Iterable, List, Union
 import torch
 import numpy as np
+from torch.types import Number
 from lightwood.encoder.base import BaseEncoder
 from lightwood.helpers.log import log
 from lightwood.helpers.general import is_none
+from lightwood.api.dtype import dtype
 
 
 class NumericEncoder(BaseEncoder):
+    """
+    The numeric encoder takes numbers (float or integer) and converts it into tensors of the form:
```

Comment:
Would be nice to explain why we choose to do it this way. 

One interesting thing that comes out of it is that you now have features that point to modalities of *why* a prediction goes the way it does, but I don't actually know the rationale of why this is the case.

Reply:
I honestly don't remember why we chose it, i.e. I mentioned the experiments we did during the call but they were inconclusive, I don't think we can motivate this until we have benchmarks to show it's good in specific scenarios.